README.md

```diff
@@ -41,124 +40,129 @@ You can send metrics, histograms, or trace data from your application to the ser
+* Verify that you have access to the CSP console. For details, see [Operations for Applications Permissions](https://docs.wavefront.com/csp_permissions_overview.html#operations-for-applications-permissions).
+* [Generating an API token](https://developer.vmware.com/apis/csp/).
 
-* The name of the host that will run the Wavefront Proxy.
-* One or more proxy listening ports to send data to. The ports you specify depend on the kinds of data you want to send (metrics, histograms, and/or trace data). You must specify at least one listener port.
-* Optional settings for tuning communication with the proxy.
-
-> **Note**: See [Advanced Proxy Configuration and Installation](https://docs.wavefront.com/proxies_configuring.html) for details.
+**Example**: Use a factory class to create a WavefrontClient and send data to Operations for Applications via CSP Api Token.
 
 ```python
-from wavefront_sdk import WavefrontProxyClient
+from wavefront_sdk.client_factory import WavefrontClientFactory
 
 # Create a sender with:
-   # the proxy hostname or address
-   # the default listener port (2878) for sending metrics to
-   # the recommended listener port (2878) for sending histograms to
-   # the recommended listener port (30000) for sending trace data.
-   # if you are directly using the sender sdk to send spans without using any other sdk, use the same port as the customTracingListenerPorts configured in the wavefront proxy for the tracing_port
-wavefront_sender = WavefrontProxyClient(
-   host="<PROXY_HOST>",
-   metrics_port=2878,
-   distribution_port=2878,
-   tracing_port=30000,
-   event_port=2878
-)
+   # Required Parameter
+   #   URL format to send data via direct ingestion: "https://<DOMAIN>.wavefront.com"
+   #   URL format to get tokens via csp authentication: "https://<CSP_ENDPOINT>.cloud.vmware.com"
+   #   CSP Api Token for csp authentication: "<CSP_API_TOKEN>"
+   # Optional Parameter
+   #   max queue size (in data points). Default: 50000
+   #   batch size (in data points). Default: 10000
+   #   flush interval (in seconds). Default: 1 second
+
+client_factory = WavefrontClientFactory()
+client_factory.add_client(
+    url="<URL for direct ingestions>",
+    csp_base_url='<URL for csp authentication>',
+    csp_api_token="<Token for csp api>",
+    max_queue_size=50000,
+    batch_size=10000,
+    flush_interval_seconds=5)
+wavefront_sender = client_factory.get_client()
 ```
 
-> **Note:** When you set up a Wavefront Proxy on the specified proxy host, you specify the port it will listen to for each type of data to be sent. The `WavefrontProxyClient` must send data to the same ports that the Wavefront Proxy listens to. Consequently, the port-related parameters must specify the same port numbers as the corresponding proxy configuration properties:
+#### Initialize the WavefrontClient (CSP OAuth App)
```

Comment:
CSP OAuth App --> Server to Server OAuth App

Initialize the WavefrontClient with a Server to Server OAuth App